src/core/qgsprovidermetadata.h

```diff
@@ -288,6 +291,16 @@ class CORE_EXPORT QgsProviderMetadata : public QObject
       const QgsCoordinateReferenceSystem &crs,
       const QStringList &createOptions = QStringList() ) SIP_FACTORY;
 
+    /**
+     * Creates mesh data source (depending of the provider)
```

Comment:
not sure what the function does from the docs, can you clarify... does it persists mesh frame to the file?

can you rename the function to createMeshFrame? not sure what data represents here?

Reply:
persisting depend of the provider. For now, only MDAL provider writes on file, but we can imagine other provider store the mesh frame with other way (in memory or other).
For the name of the method, it is inspired, to keep consistence,  from `QgsMeshDataSourceInterface`, one of the base class of `QgsMeshDataProvider`.

Reply:
ok, maybe clarify docs to mention is is mesh frame storage